autoware_adapi_v1_msgs/vehicle/msg/VehicleSignal.msg

```diff
@@ -0,0 +1,9 @@
+# constants
+uint8 DISABLE = 1
+uint8 ENABLE_LEFT = 2
+uint8 ENABLE_RIGHT = 3
+uint8 HAZARD_ENABLE = 2
+
+builtin_interfaces/Time stamp
+uint8 turn_indicator
+uint8 hazard_light
```

Comment:
I think it's easier to understand if the messages are separated by `turn_indicator` and `hazard_light`. And how about merging with gear.

Reply:
> I think it's easier to understand if the messages are separated by turn_indicator and hazard_light. And how about merging with gear.

Yes I agree, we can combine with gear and use the name vehicle state

Reply:
changed at [9babd45](https://github.com/autowarefoundation/autoware_adapi_msgs/pull/24/commits/9babd4550fcbc8f50e1ffcadc5f9cb9813ce921b)